lib/api/slack.ex

```diff
@@ -16,12 +18,44 @@ defmodule Aprb.Api.Slack do
       requires :response_url, type: String
     end
     post do
+      # check that token matches, that the POST comes from our slack integration
+      if System.get_env("SLACK_SLASH_COMMAND_TOKEN") != params[:token] do
+        conn
+          |> put_status(403)
+          |> json(%{message: "Unauthorized"})
+          |> halt()
+      end
 
-      # TODO: check that token matches, that the POST comes from our slack integration
-      # if System.get_env("SLACK_SLASH_COMMAND_TOKEN") != params["token"] ...
+      subscriber = case Repo.get_by(Subscriber, user_id: params[:user_id]) do
```

Comment:
It's a different feature to subscribe to something for yourself and be DMed the subscription - I think that will work just out of the box if you DM the bot, because the channel ID represents a private channel between the bot and the user.